webnn/resources/test_data/div.json

```diff
@@ -0,0 +1,784 @@
+{
+  "tests": [
+    {
+      "name": "div two float32 1D tensors",
+      "inputs": {
+        "a": {
+          "shape": [24],
+          "data": [
+            60.42374184854583,
+            -86.92247771592565,
+            -19.496113538677307,
+            -15.150616021872224,
+            13.455190756956796,
+            45.4335994568701,
+            61.08286131089616,
+            70.71882919883615,
+            -31.27858011424047,
+            56.083543107783214,
+            38.99276571709217,
+            -3.275369300234928,
+            32.289323532099786,
+            -3.676541725498467,
+            88.4349140505588,
+            14.369060854798647,
+            13.94319409117783,
+            16.861190475721344,
+            4.816806540352701,
+            44.15916607007162,
+            -13.083211898368276,
+            44.565997906464105,
+            -34.892785072690245,
+            -74.09375203216162
+          ],
+          "type": "float32"
+        },
+        "b": {
+          "shape": [24],
+          "data": [
+            -95.02903333542001,
+            62.804865007594174,
+            -85.32865597192423,
+            -68.20919717822437,
+            79.45568261298038,
+            -68.69049322917093,
+            -94.46466465496823,
+            -10.000411029009541,
+            18.318865013229242,
+            -3.623280011603569,
+            -5.957828048196006,
+            89.49882849628409,
+            94.95791079054848,
+            -79.00059056550418,
+            -79.87597038424425,
+            74.997874802068,
+            25.86537368134097,
+            91.54433121797925,
+            81.65287344691947,
+            48.214831959279195,
+            63.37012091471408,
+            10.62638472655793,
+            46.12662564540156,
+            77.2232730204774
+          ],
+          "type": "float32"
+        }
+      },
+      "expected": {
+        "name": "ouput",
+        "shape": [24],
+        "data": [
+          -0.6358450651168823,
+          -1.3840086460113525,
+          0.22848258912563324,
+          0.22211983799934387,
+          0.16934208571910858,
+          -0.6614248752593994,
+          -0.6466212868690491,
+          -7.071591854095459,
+          -1.7074518203735352,
+          -15.478666305541992,
+          -6.544795513153076,
+          -0.036596786230802536,
+          0.3400382697582245,
+          0.046538159251213074,
+          -1.1071529388427734,
+          0.19159291684627533,
+          0.5390679836273193,
+          0.18418607115745544,
+          0.058991268277168274,
+          0.9158834218978882,
+          -0.20645710825920105,
+          4.193900108337402,
+          -0.7564564943313599,
+          -0.9594743251800537
+        ],
+        "type": "float32"
+      }
+    },
+    {
+      "name": "div two float32 2D tensors",
```

Comment:
[minor] Propose deleting "two", because can the input count ever not be two? (so doesn't really add more information to the test name) Plus, you didn't use "two" below for other names "div float32 broadcast 2D to 4D".

Reply:
Thanks. Updated.